src/core/handshaker/handshaker.cc

```diff
@@ -125,24 +135,21 @@ void HandshakeManager::DoHandshake(
 void HandshakeManager::Shutdown(absl::Status error) {
   MutexLock lock(&mu_);
   if (!is_shutdown_) {
-    GRPC_TRACE_LOG(handshaker, INFO)
-        << "handshake_manager " << this << ": Shutdown() called: " << error;
+    GRPC_CHANNELZ_LOG(args_.trace_node) << "Shutdown called: " << error;
     is_shutdown_ = true;
     // Shutdown the handshaker that's currently in progress, if any.
     if (index_ > 0) {
-      GRPC_TRACE_LOG(handshaker, INFO)
-          << "handshake_manager " << this
-          << ": shutting down handshaker at index " << index_ - 1;
+      GRPC_CHANNELZ_LOG(args_.trace_node)
+          << "Shutting down handshaker at index " << (index_ - 1);
       handshakers_[index_ - 1]->Shutdown(std::move(error));
     }
   }
 }
 
 void HandshakeManager::CallNextHandshakerLocked(absl::Status error) {
-  GRPC_TRACE_LOG(handshaker, INFO)
-      << "handshake_manager " << this << ": error=" << error
-      << " shutdown=" << is_shutdown_ << " index=" << index_
-      << ", args=" << HandshakerArgsString(&args_);
+  GRPC_CHANNELZ_LOG(args_.trace_node)
+      << "error=" << error << " shutdown=" << is_shutdown_
```

Comment:
This message should probably start with something like "CallNextHandshakerLocked:", so that it's clear what it's about from the channel trace, where there are no line numbers.

Reply:
agree... moving it out of channelz also (because I don't think it makes sense there)